plugins/org.jkiss.dbeaver.tasks.native/src/org/jkiss/dbeaver/tasks/nativetool/AbstractNativeToolHandler.java

```diff
@@ -175,9 +175,18 @@ protected boolean validateTaskParameters(DBTTask task, SETTINGS settings, Log lo
         return true;
     }
 
-    abstract protected java.util.List<String> getCommandLine(SETTINGS settings, PROCESS_ARG arg) throws IOException;
+    abstract protected java.util.List<String> getCommandLine(
+        SETTINGS settings,
+        PROCESS_ARG arg,
+        @NotNull DBRProgressMonitor monitor
```

Comment:
Just a nitpick: I'd pass the monitor as the very first argument.

Reply:
Usually, I do this way, yes. But here are generics. I really think that they need to be first.